episodes/05-counting-mining.md

```diff
@@ -649,6 +649,16 @@ $ grep -iw hero *a.tsv > results/hero-i.tsv
 
 ::::::::::::::::::::::::::::::::::::::::::::::::::
 
+:::::::::::::::::::::::::::::::::::::::::::::::::: instructor
+
+### Watch out for different behaviour of the -E option across systems
+
+Learners may encounter some inconsistent behaviour between operating systems when solving the challenge below.
```

Comment:
Since this is referring to the next section, it would make sense to move this into that section (as an H3 under the next H2). Right now this positions it as a heading under the previous solution that it isn't related to.

Reply:
Thanks for the review @kaitlinnewson. To make sure that I understand correctly: you would prefer the instructor note to appear inside the challenge block it refers to, rather than above it?